### PR TITLE
patch: update organisation users

### PR DESCRIPTION
**Pull request by DONOTFORGET-BYTE**

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 19 min

**Comment by DONOTFORGET-BYTE** [comment]
@dimasciput @danangmassandy I made a few changes Amy spotted a few issues

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 19 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 19 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 10 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 20 min

File: django_project/sawps/views.py [hunk 1]
```diff
@@ -140,27 +140,37 @@ def send_invitation_email(self, email_details):
...
+                    'name': name,
+                    'request_message': request_message
                 },
             )
             send_mail(
                 subject,
                 None,
                 settings.SERVER_EMAIL,
-                ['to@gmail.com'],
+                ['amy@kartoza.com'],
```
Comment (dimasciput) [inline django_project/sawps/views.py:167]: don't put the email directly here. Instead, use the variable from the Django settings 

https://github.com/kartoza/sawps/blob/eb33c7ff255d8f7dc4a974d79f88c2525bf4c6f6/django_project/core/settings/contrib.py#L106

File: django_project/sawps/views.py [hunk 2]
```diff
@@ -140,27 +140,37 @@ def send_invitation_email(self, email_details):
...
                 html_message=message
             )
             return JsonResponse({'status': 'success'})
         except Exception as e:
                 print('Failed to send email:', str(e))
```
Comment (dimasciput) [inline django_project/sawps/views.py:172]: remove this

**Review by dimasciput — CHANGES_REQUESTED** [review]
Please clean up your PR, it currently displays old commits from a different PR.

**Comment by DONOTFORGET-BYTE** [comment]
@dimasciput I have used the support email from contrib.py

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 12 min

**Comment by dimasciput** [comment]
Screenshot 2023-07-14 at 1 18 56 PM

@DONOTFORGET-BYTE your PR still displays old commits

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 56 sec

**Pull request closed without merging**